src/object-properties.js

```diff
@@ -37,26 +37,108 @@ const properties = {
      * @type {string}
      */
     label: 'Button',
+    /**
+     * @type {object}
+     */
+    font: {
+      /**
+       * @type {string}
+       */
+      size: '',
+      /**
+       * @type {boolean}
+       */
+      useColorExpression: false,
+      /**
+       * @type {object}
+       */
+      color: {
+        index: -1,
+        color: null,
+      },
+      /**
+       * @type {string}
+       */
+      colorExpression: '',
+      /**
+       * @type {object}
+       */
+      style: {
+        bold: true,
+        italic: false,
+        underline: false,
+      },
+      /**
+       * @type {string}
+       */
+      align: 'center',
+    },
+    /**
+     * @type {object}
+     */
+    background: {
+      /**
+       * @type {boolean}
+       */
+      useColorExpression: false,
+      /**
+       * @type {object}
+       */
+      color: {
+        index: -1,
+        color: null,
+      },
+      /**
+       * @type {string}
+       */
+      colorExpression: '',
+      /**
+       * @type {boolean}
+       */
+      useImage: false,
+      /**
+       * @type {object}
+       */
+      url: {
+        /**
+         * @type {object}
+         */
+        qStaticContentUrlDef: {
+          /**
+           * @type {string}
+           */
+          qUrl: '',
+        },
+      },
+      /**
+       * @type {string}
+       */
+      size: 'auto',
+      /**
+       * @type {string}
+       */
+      position: 'topLeft',
```

Comment:
I think a center position as default for the image would be better actually. Maybe discuss with Christofer

Reply:
sure, I'll change